### PR TITLE
Updated changes for 1.5.0 release

### PR DESCRIPTION
**Pull request by sraghunandan**

### CODE REVIEW

**Comment by sraghunandan** [comment]
@sgururajshetty please review this

File: content/WEB-INF/classes/application.conf [hunk 1]
```diff
@@ -16,7 +16,8 @@ fileList=["configuration-parameters",
   "release-guide",
   "how-to-contribute-to-apache-carbondata",
   "introduction",
-  "usecases"
+  "usecases",
+  "CSDK-guide"
```
Comment (sgururajshetty) [inline content/WEB-INF/classes/application.conf]: missing file
> carbon-as-spark-datasource-guid

**Review by sgururajshetty — CHANGES_REQUESTED** [review]

**Comment by sraghunandan** [comment]
handled the comment @sgururajshetty

**Comment by sgururajshetty** [comment]
LGTM

**Pull request merged**